### PR TITLE
CI: Test SciPy ASR with every commit 

### PR DESCRIPTION
**Pull request by Pranavchiku**
TODO:
* [ ] merge the PR
* [ ] send a test PR that does not generate ASR and ensures the CI breaks.

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -604,6 +604,19 @@ jobs:
...
+            cd ..
+            rm -rf scipy/
+            git clone https://github.com/pranavchiku/scipy.git
+            cd scipy
+            git checkout -t origin/scipy1
+            git checkout a70f11a305edd024cb9a0e5048257eb0f51e29e1
+            find . -name "*.f" > files.txt
+            files=$(<files.txt)
+            for file in $files; do
+                ../src/bin/lfortran --fixed-form --implicit-typing --implicit-interface --implicit-argument-casting --show-asr $file > $file.f90
```
Comment (certik) [inline .github/workflows/CI.yml:616]: You can turn off colors:
```suggestion
                ../src/bin/lfortran --fixed-form --implicit-typing --implicit-interface --implicit-argument-casting --show-asr --no-color $file > $file.f90
```

**Review by certik — APPROVED** [review]
I think that looks good. You can do that `--no-color` thing in another PR, to save on CI running time.

**Comment by certik** [comment]
@Pranavchiku let's merge this?

**Comment by Pranavchiku** [comment]
Yes, I will apply code review just a minute

**Comment by certik** [comment]
I would apply the code review in a new PR.

**Pull request merged**